### PR TITLE
Add notes on external snippets plugins

### PR DESCRIPTION
**Pull request by RayZ0rr**
* Added some clarity to readme
* Fixed some broken links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,10 +163,14 @@ from the nvim-cmp wiki.
 ## Add Snippets
 To test if LuaSnip works correctly and see some of the capabilities lua-native snippets have, `:luafile` `Examples/snippets.lua`.  
 To add snippets in lua, add them to the [`require'luasnip'.snippets`-table](https://github.com/L3MON4D3/LuaSnip/blob/master/Examples/snippets.lua#L167).  
-To load snippets from a package that provides snipmate-like (eg. [honza/vim-snippets](https://github.com/honza/vim-snippets)) or vscode-like ([rafamadriz/friendly-snippets](https://github.com/rafamadriz/friendly-snippets)) snippets, check [this](https://github.com/L3MON4D3/LuaSnip/blob/5eb67644205386bd02b93a7b570293f827f58e4b/Examples/snippets.lua#L495) section of the examples.
+To load snippets from a package that provides snippets :-
```
Comment (L3MON4D3) [inline README.md:166]: Imo "Luasnip is also capable of loading snippets from packages:" sounds a bit better

**Comment by L3MON4D3** [comment]
Could you also link the `Examples/snippets.lua`-section about loading? It's very far down and thus not very visible, and might be just enough to get started with.

File: README.md (hunk 1 above)
Comment (L3MON4D3) [reply README.md:166]: (nitpick :D)

Comment (RayZ0rr) [reply README.md:166]: Okay will do this

**Comment by L3MON4D3** [comment]
Thanks for your contribution :)

**Pull request merged**